### ISSUE TITLE
Modification Advice on 'vemit-child' Snippet

### ISSUE THREAD

**Issue by Keikana** [sdras/vue-vscode-snippets#114]
The original snippet was:
```
@change="emit('change', event.target.value)"
```
I guess you are trying to type `$emit` but the `$` was recognized as a symbol for placeholder. You can use `\\$` in the snippet body instead.

**Comment by itpropro** [comment]
This is correct syntax. In Vue 3 you can define emits with the `defineEmits` compiler macro. E.g.:

```
<script setup lang="ts">
const emit = defineEmits<{ (event: 'eventName', id: string): void }>()
</script>
<template>
    <button
      @click="emit('eventName', 'abc')">
    </button>
</template>
```

You can find more information here: https://vuejs.org/guide/typescript/composition-api.html#typing-component-emits
I would keep it this way, as `$emit` would probably only be used with the options API that is not the recommended default anymore with Vue 3.
